### PR TITLE
improving Dockerfile, less images to complete the docker build

### PR DESCRIPTION
**Pull request by nhatfield**

### CODE REVIEW

**Comment by TheConnMan** [comment]
This project now has a `dev` branch you can edit this PR to be against.

**Comment by nhatfield** [comment]
PR base switched from Master to Dev

File: build/Dockerfile.rpm [hunk 1]
```diff
@@ -6,21 +6,24 @@
...
+RUN yum -y update \
+ && yum groupinstall -y development \
+ && yum -y install fakeroot mysql mysql-devel postgresql-devel lm_sensors-devel cmake screen which tar wget \
+ && rpm -e git perl-Git \
+ && yum -y install curl-devel expat-devel gettext-devel openssl-devel zlib-devel \
+ && yum -y install gcc perl-ExtUtils-MakeMaker \
+ && wget https://www.kernel.org/pub/software/scm/git/git-2.9.2.tar.gz \
+ && tar xzf git-2.9.2.tar.gz \
+ && rm -f git-2.9.2.tar.gz \
+ && yum clean all
```
Comment (yahonggu) [inline build/Dockerfile.rpm]: do we want to add 'apt-get clean' for debian:7 since we added 'yum clean all' here.

Comment (nhatfield) [reply build/Dockerfile.rpm]: good point, i also saw a couple more that could use some cleanup. adding them now...

**Pull request merged**